esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1084,7 +1140,9 @@ unsigned WsWuInfo::getWorkunitThorLogInfo(IArrayOf<IEspECLHelpFile>& helpers, IE
             uniqueProcesses.setValue(processName.str(), true);
 
             StringBuffer groupName;
-            getClusterThorGroupName(groupName, processName.str());
+            cw->getThorGroup(groupName);
```

Comment:
see other comment.
I know I suggested it, but as you can see now you've implemented, it doesn't make sense, because the group can be different depending on 'processName', i.e. Thor can run on >1 group.
Need to remove getThorGroup/setThorGroup.

Reply:
Will remove

Reply:
Can one WU use >1 thor group?

Reply:
yes.
A workunit can run on any of the Thor's attached to the queue, and there can be any number of Thors, which may be on same group, or independent groups.

Reply:
in fact I was wrong, all Thor's in a logical cluster must be the same size *at the moment*.
It would be best to try not to assume that's true though, as it may well change.